### DISCUSSION TITLE
blog/2023/12/11/separate-bibliographies-quarto/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#99] [Blog comments]
# blog/2023/12/11/separate-bibliographies-quarto/

Use multibib and Quarto to create separate bibliographies for a document and an appendix

https://www.andrewheiss.com/blog/2023/12/11/separate-bibliographies-quarto/

**Comment by RobWiederstein** [comment]
Thanks, Andrew, for the helpful post. I'm trying to figure out "multibib" now and was happy to find your post! --rob

**Comment by IAFomichev** [comment]
Thank you for this great and useful post. Do you know if one can change citation style when using multibib?

**Reply by IAFomichev** [reply]
Ended up just using a custom .csl, worked just fine.

**Comment by julienleroux5** [comment]
Nice work! Do you think it could be possible to apply two different styles to the two bibliographies and corresponding citations? (e.g. with different .csl files or something else in biblatex) for example to get one section with citations and bibliography with numbering, and the other one ordered by author name and citations with authoryear style?
